zerver/models.py

```diff
@@ -989,10 +992,15 @@ def get_realm_emoji_dicts(
         if realm_emoji.author:
             author_id = realm_emoji.author_id
         emoji_url = get_emoji_url(realm_emoji.file_name, realm_emoji.realm_id)
+        still_url = ""
```

Comment:
What's the thinking on the choice of `""` vs. `None` (or omitting the field) for non-animated images?  Might be a good api design thread.

Reply:
Looks like we decided to go with omitting the field, which seems like a good choice.